tests/st/utils/utils.py

```diff
@@ -168,8 +168,20 @@ def assert_output_contains(self, text):
             "command=" + self.command + "\noutput=\n" + self.output + \
             "\nexpected=\n" + text
 
+    def assert_output_not_contains(self, text):
+        """
+        Assert the calicoctl command output does not contain the supplied text.
+        Args:
+            text:   Expected text in the command output.
+        """
+        if not text:
+            return
+        assert not text in self.output, "Unxpected text in output; \n" + \
+            "command=" + self.command + "\noutput=\n" + self.output + \
+            "\nunexpected=\n" + text
```

Comment:
As above, I suggest adding this check in `assert_error`, instead of creating and calling this new method.

Reply:
we can do the check for no panic in the assert_error for sure. If there was a panic, the assertion would fail.

Reply:
done